pages/mentoring_faqs.md

```diff
@@ -9,11 +9,12 @@ If you're not sure what feedback to give, paste a link to the solution (e.g. `ht
 ### How can I improve my mentoring skills?
 This is something we're extremely passionate about helping you with and we are in the process of writing some awesome guides on giving great feedback with lots of examples. Eventually we hope to have a guide for each track.
 
-In the meantime, here's some recommended reading:
+In the meantime, here's some recommended resources:
 
 - [The Science and Art of Receiving Feedback Well](https://www.amazon.com/Thanks-Feedback-Science-Receiving-Well-ebook/dp/B00DMCV0XE/)
 - [Unlearning Toxic Behaviours in a Code Review Culture - Conference talk by Sandya Sankarram](https://drive.google.com/file/d/1ROZA9yKpEa8suiqVXtPO0IJtqa95SOjE/view) and related [Blog Post](https://medium.freecodecamp.org/unlearning-toxic-behaviors-in-a-code-review-culture-b7c295452a3c)
 - [How Positive and Negative Feedback Motivate Goal Pursuit](https://faculty.chicagobooth.edu/ayelet.fishbach/research/FEF%20Compass%202010.pdf)
+- The #mentor-the-mentor channel on Slack, where you can learn to be an amazing mentor from other amazing mentors.
```

Comment:
```suggestion
- The [#mentor-the-mentor channel](https://exercism-team.slack.com/messages/CM3SY1F2L) on Slack, where you can learn to be an amazing mentor from other amazing mentors.
```

Reply:
Does this link work for others?

Reply:
Not opposed, but we don't link to any of the other channels in these documents - should we got and add links to them all?